src/main/java/io/github/thebusybiscuit/hotbarpets/groups/PeacefulAnimals.java

```diff
@@ -48,8 +48,16 @@ public void load(HotbarPets plugin) {
 				new ItemStack(Material.SALMON), new ItemStack(Material.EMERALD), new ItemStack(Material.SALMON),
 				new ItemStack(Material.LAPIS_LAZULI), new ItemStack(Material.COD), new ItemStack(Material.LAPIS_LAZULI)
 		}).register();
-	}
-
-
 
+		new HotbarPet(plugin.getCategory(), new SlimefunItemStack(
+				"HOTBAR_PET_PANDA",
+				"414ff627a6a6f35e1d717ebcb191e4c7f9097542db599e7108ae2c7dd3513e51",
+				"&8Panda &fPet",
+				getName(),"&7Favorite Food: Bamboo", "", "&rScares away phantoms"),
```

Comment:
I'd rather write "This sleepy Panda protects you from Insomnia", "Phantoms will no longer chase you at night"
instead of this ambigous "Scares away phantoms"